include/nbl/asset/utils/ICPUVirtualTexture.h

```diff
@@ -132,8 +132,12 @@ class ICPUVirtualTexture final : public IVirtualTexture<ICPUImageView, ICPUSampl
 
         upscaled_img->setBufferAndRegions(std::move(buf), std::move(regions));
 
-        using blit_filter_t = asset::CBlitImageFilter<asset::VoidSwizzle,asset::IdentityDither/*TODO: White Noise*/,void,false,asset::CMitchellImageFilterKernel<>>;
-        blit_filter_t::state_type blit;
+        using blit_filter_t = asset::CBlitImageFilter<asset::VoidSwizzle,asset::IdentityDither/*TODO: White Noise*/,void,false, asset::CBlitUtilities<asset::CWeightFunction1D<asset::SMitchellFunction<>>>>;
```

Comment:
style nitpick, you can remove `asset::` from all of the types, you're already in this namespace

Reply:
Resolved.